Simplenote/Information/NoteInformationViewController.swift

```diff
@@ -177,6 +179,20 @@ private extension NoteInformationViewController {
 
         additionalSafeAreaInsets = UIEdgeInsets(top: headerStackView.frame.maxY, left: 0, bottom: 0, right: 0)
     }
+    
+    func configureDragBar() {
+        if navigationController == nil {
+            dragBar.translatesAutoresizingMaskIntoConstraints = false
+            NSLayoutConstraint.activate([
+                dragBar.heightAnchor.constraint(equalToConstant: 5),
+                dragBar.widthAnchor.constraint(equalToConstant: 36),
+                dragBar.topAnchor.constraint(equalTo: view.topAnchor, constant: 5),
+                dragBar.centerXAnchor.constraint(equalTo: view.centerXAnchor)
+            ])
+        } else {
+            dragBar.isHidden = true
+        }
```

Comment:
Since the initialization is already done in IB (`NoteInformationViewController`), would it be possible to move the Autolayout constraints there?

---

On a second note, we'd typically wanna avoid inline constants. Instead, you would probably wanna do something like this:

```
   dragBar.heightAnchor.constraint(equalToConstant: Metrics.dragBarSize.height),
   dragBar.widthAnchor.constraint(equalToConstant: Metrics.dragBarSize.width),
   dragBar.topAnchor.constraint(equalTo: view.topAnchor, constant: Metrics.dragBarTopPadding),
   dragBar.centerXAnchor.constraint(equalTo: view.centerXAnchor)
```

And at the bottom of the source file you could have:

```
private enum Metrics {
    static let dragBarSize = CGSize(width: 36, height: 5)
    static let dragBarTopPadding = CGFloat(5)
}
```

### Why:
Numbers used inline in the code are best avoided, for few reasons. It's harder to maintain, it's impossible to reuse. Suppose there were two components onscreen that needed the same **Top Padding**, without a constant, you'd have a `5` in two places.

**Plus:** if you had to change that `5`, without a proper constant you'd need to read the whole code, and patch two spots.

### And another note:
Why `enum Metrics` instead of, say, `struct Metrics`?  Because if you used `struct Metrics` this statement would be valid:

```
let metrics = Metrics()
```

It's a "workaround" we've been using everywhere, and allows us to namespace a bunch of constants. Other programming languages (such as C#) have a way to define a namespace that doesn't really belong to anything that can be instantiated. Swift... really doesn't, for now!